libnetwork/netutils/utils_linux_test.go

```diff
@@ -40,29 +40,6 @@ func TestOverlappingNameservers(t *testing.T) {
 	}
 }
 
-func TestCheckRouteOverlaps(t *testing.T) {
-	networkGetRoutesFct = func(netlink.Link, int) ([]netlink.Route, error) {
-		routesData := []string{"10.0.2.0/32", "10.0.3.0/24", "10.0.42.0/24", "172.16.42.0/24", "192.168.142.0/24"}
-		routes := []netlink.Route{}
-		for _, addr := range routesData {
-			_, netX, _ := net.ParseCIDR(addr)
-			routes = append(routes, netlink.Route{Dst: netX})
-		}
-		return routes, nil
-	}
-	defer func() { networkGetRoutesFct = nil }()
-
-	_, netX, _ := net.ParseCIDR("172.16.0.1/24")
-	if err := CheckRouteOverlaps(netX); err != nil {
-		t.Fatal(err)
-	}
-
-	_, netX, _ = net.ParseCIDR("10.0.2.0/24")
-	if err := CheckRouteOverlaps(netX); err == nil {
-		t.Fatal("10.0.2.0/24 and 10.0.2.0 should overlap but it doesn't")
-	}
-}
-
 func TestCheckNameserverOverlaps(t *testing.T) {
```

Comment:
I _think_ all the other tests should actually be ok on other platforms, so reverting the rename of this file and applying the change I had in https://github.com/moby/libnetwork/pull/2590.

If that explodes, I'll fix it

Reply:
Ah, some other tests also use `netns`. Let me just drop the first commit